tests/util/test_full_block_utils.py

```diff
@@ -217,25 +224,26 @@ def get_full_blocks() -> Iterator[FullBlock]:
                         for reward_chain_sp_proof in [vdf_proof(), None]:
                             for infused_challenge_chain_ip_proof in [vdf_proof(), None]:
                                 for finished_sub_slots in get_finished_sub_slots():
-                                    yield FullBlock(
-                                        finished_sub_slots,
-                                        reward_chain_block,
-                                        challenge_chain_sp_proof,
-                                        vdf_proof(),  # challenge_chain_ip_proof
-                                        reward_chain_sp_proof,
-                                        vdf_proof(),  # reward_chain_ip_proof
-                                        infused_challenge_chain_ip_proof,
-                                        foliage,
-                                        foliage_transaction_block,
-                                        transactions_info,
-                                        generator,  # transactions_generator
-                                        [],  # transactions_generator_ref_list
-                                    )
+                                    for refs_list in get_ref_list():
+                                        yield FullBlock(
+                                            finished_sub_slots,
+                                            reward_chain_block,
+                                            challenge_chain_sp_proof,
+                                            vdf_proof(),  # challenge_chain_ip_proof
+                                            reward_chain_sp_proof,
+                                            vdf_proof(),  # reward_chain_ip_proof
+                                            infused_challenge_chain_ip_proof,
+                                            foliage,
+                                            foliage_transaction_block,
+                                            transactions_info,
+                                            generator,  # transactions_generator
+                                            refs_list,  # transactions_generator_ref_list
+                                        )
 
 
 @pytest.mark.asyncio
-@pytest.mark.skip("This test is expensive and has already convinced us the parser works")
-async def test_parser(self):
+# @pytest.mark.skip("This test is expensive and has already convinced us the parser works")
```

Comment:
Is this intentional? Is this test even slower now?

Reply:
yes. in my first incarnation of this patch I had a bug that wasn't caught by this test (because it didn't cover the ref-list). So, regrettably, this test now takes longer. But we can probably disable it again after this lands. It's here now to demonstrate that the parser works correctly